Jenkinsfile

```diff
@@ -19,6 +20,7 @@ pipeline {
     disableConcurrentBuilds()
   }
   parameters {
+    booleanParam(name: 'create_db', defaultValue: false, description: 'Create New MongoDB Environment')
```

Comment:
We should provably default this to true because that will be the common case

Reply:
This should be totally fixed now.  Defaults should all be set to true.